packages/cra-template-mephisto-review/template/src/GridView.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import { Card, Elevation, H4 } from "@blueprintjs/core";
+
+function GridView({ data }) {
+  const history = useHistory();
+
+  return data && data.length > 0 ? (
+    <div className="grid-container">
+      {data.map((item) => (
+        <Card
+          key={item.id}
+          interactive={true}
+          elevation={Elevation.TWO}
+          className="grid-item"
+          onClick={() => history.push(`/${item.id}`)}
```

Comment:
any specific reason for using `history.push` here as opposed to wrapping the Card in a `<Link />` tag? Just curious

Reply:
I forgot that was an available component, I will change that